app/src/conf/config.rinkeby.json

```diff
@@ -3,6 +3,8 @@
   "markets": [
     {
       "questionId": "0xda66c02003000000000000000000000000000000000000000000000000000000",
+      "dataSource": "DataSource",
+      "dataSourceUrl": "https://example.org",
```

Comment:
I think it should be taking it from the API right? so this is no longer needed

Reply:
Not for now, as we're using the config again for a moment to hardcode the scalar properties.